baseline/OSPS-GV.yaml

```
@@ -12,8 +12,17 @@ criteria:
       The project documentation MUST include the
       Roles and Responsibilities for members of the
       project.
-    rationale: # TODO
-    implementation: # TODO
+    rationale: |
+      Documenting project roles and responsibilities
+      helps project particpants, potential contributors,
+      and downstream consumers have an accurate
+      understand of who is working on the project
+      and what areas or authority they may have.
```

```suggestion
      and what areas of authority they may have.
```
I assume this is what you meant?